apps/database/main.py

```
@@ -1,177 +1,141 @@
-        )
+        await logs.insert_one({
+            "name": data.name,
+            "error": str(e),
+            "created_at": get_utc_now(),
+            "status": "error",
+            "error_id": error_id,
+            "type": "insert_error",
+        })
         raise HTTPException(
             status_code=500, detail=f"Error during database insertion: {str(e)}"
         )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)